test/e2e/petset.go

```diff
@@ -67,7 +67,7 @@ const (
 // Time: 25m, slow by design.
```

Comment:
Which of these tests are actually slow? I'm not sure that they take more than 5 mins to run on average.